src/initiatives/index.ts

```diff
@@ -0,0 +1,113 @@
+import "@polkadot/api-augment/kusama";
+import { WsProvider, ApiPromise } from "@polkadot/api";
+import { blake2AsHex } from '@polkadot/util-crypto';
+
+import { Signer } from "@polkadot/api/types";
+import { AccountId, AccountMembership, Conviction, TreasuryPeriod, VoteOpenGov } from "../types";
+import { initiativeRemoveMembers } from "./remove-members";
+import { initiativeAddMembers } from "./add-members";
+import { initiativeTreasuryRequest } from "./treasury";
+import { initiativeConvictionVoting } from "./conviction-voting";
+
+export async function topupThenInitiativeSetup(
```

Comment:
Indeed. This shouldn't be called `topupThen...` as it doesn't top up anything.

```suggestion
export async function initiativeSetup(
```